src/lib/lightnet.js

```diff
@@ -0,0 +1,723 @@
+ * @param {string}  argv.cleanUp - Whether to perform the clean up.
+ * @param {boolean} argv.debug - Whether to print the debug information.
+ * @returns {Promise<void>}
+ */
+export async function stopLightnet({ saveLogs, cleanUp, debug }) {
+  isDebug = debug;
+  let logsDir = null;
+  await checkDockerEngineAvailability();
+  await step('Checking prerequisites', async () => {
+    await handleStopCommandChecks(lightnetDockerContainerName);
+  });
+  await step(
+    'Stopping the lightweight Mina blockchain network Docker container',
+    async () => {
+      await stopDockerContainer(lightnetDockerContainerName);
+    }
+  );
+  if (saveLogs && fs.existsSync(lightnetConfigFile)) {
+    await step('Preserving Docker container processes logs', async () => {
+      logsDir = await saveDockerContainerProcessesLogs();
+    });
+  }
+  if (cleanUp) {
+    await step('Cleaning up', async () => {
+      await removeDockerContainer(lightnetDockerContainerName);
+      await removeDanglingDockerImages();
+      if (fs.existsSync(lightnetConfigFile)) {
+        await removeDockerVolume(
+          fs.readJSONSync(lightnetConfigFile).containerVolume
+        );
+      }
+      await fs.remove(lightnetConfigFile);
+    });
+  }
+  if (logsDir) {
+    const boldLogs = chalk.reset.bold('logs');
+    console.info(
+      `\nThe Docker container processes ${boldLogs} can be found at:\n\n` +
+        chalk.green.bold(logsDir) +
+        '\n'
+    );
+    console.info('Done\n');
+  } else {
+    if (
+      fs.existsSync(lightnetLogsDir) &&
+      fs.readdirSync(lightnetLogsDir).length === 0
+    ) {
+      fs.removeSync(lightnetLogsDir);
+    }
+    console.info('\nDone\n');
+  }
+}
+
+/**
+ * Gets the lightweight Mina blockchain network status.
+ * @param {boolean} object.preventDockerEngineAvailabilityCheck - Whether to prevent the Docker Engine availability check.
+ * @param {boolean} object.debug - Whether to print the debug information.
+ * @returns {Promise<void>}
+ */
+export async function lightnetStatus({
+  preventDockerEngineAvailabilityCheck = false,
+  debug,
+} = {}) {
+  isDebug = debug;
+  if (!preventDockerEngineAvailabilityCheck) {
+    await checkDockerEngineAvailability();
+  }
+  const containerState = getDockerContainerState(lightnetDockerContainerName);
+  if (DockerContainerState.NOT_FOUND === containerState) {
+    console.log(
+      chalk.red(
+        '\nThe lightweight Mina blockchain network Docker container does not exist!'
+      )
+    );
+    shell.exit(1);
+  }
+  console.log('\n' + chalk.reset.bold('Lightweight Mina Blockchain Network'));
+  if (
+    DockerContainerState.RUNNING === containerState &&
+    fs.existsSync(lightnetConfigFile)
+  ) {
+    printUsefulUrls();
+    printDockerContainerProcessesLogPaths();
+    await printBlockchainNetworkProperties();
+    printZkAppSnippet();
+    printExtendedDockerContainerState(lightnetDockerContainerName);
```

Comment:
We could add links to docs here when we have them.

Reply:
By "here" you mean zkapp-cli repo readme or https://docs.minaprotocol.com/zkapps or maybe something else?
Would you prefer **skip** printing some of the info upon `zk lightnet status` execution once corresponding info will be documented?

Reply:
I meant we can display a link to docs about the lightnet network around here in the future when we have them.

Reply:
https://github.com/o1-labs/zkapp-cli/issues/519